test/integration/test_gcm_cipher.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'common'
+require 'fileutils'
+require 'tmpdir'
+
+require 'net/ssh'
+
+require 'timeout'
+
+# see Vagrantfile,playbook for env.
+# we're running as net_ssh_1 user password foo
+# and usually connecting to net_ssh_2 user password foo2pwd
+class TestGcmCipher < NetSSHTest
+  include IntegrationTestHelpers
+
+  def run_with_only_cipher(cipher)
+    config_lines = File.read('/etc/ssh/sshd_config').split("\n")
+    config_lines = config_lines.map do |line|
+      if line =~ /^Ciphers/
+        "##{line}"
+      else
+        line
+      end
+    end
+    config_lines.push("Ciphers #{cipher}")
```

Comment:
The CI failures _suggest_ to me that the ssh server is failing to start which generates a pid, since the config file is derived from the tempfile which the config_lines are written to. I've not tested this but from an eyeball, looks to be due to writing the cipher as `aes128-gcm` where a valid cipher configuration for ssh_config is `aes128-gcm@openssh.com`

Reply:
Yep the log file shows:

```sh
Bad SSH2 cipher spec 'aes128-gcm'.
```

Reply:
May I ask you  where did you find this log line ?

Reply:
If you pass the `debug: True` flag to `start_sshd_7_or_later` then it'll print the log path of that's the sshd. I'll also add a PR to tail the log file if sshd fails to start: 
https://github.com/net-ssh/net-ssh/pull/908/commits/20dd0f4029195c58c3c1178781cb491f0b10f881